conformance/tests/enums_member_values.py

```diff
@@ -77,6 +77,9 @@ class Color3(Enum):
     RED = 1  # OK
     GREEN = "green"  # E
 
+    def __init__(self, value: int) -> None:
```

Comment:
This is incorrect. The `_value_` field is special for enums. It is filled in by the "magic" logic of the EnumMeta metaclass. It should never be initialized directly by a user-provided `__init__` method. I think the `__init__` should be removed in this case. Pyre should include special-cased handling of `_value_` and avoid emitting a diagnostic in this case.